### DISCUSSION TITLE
关于模版项目Server-Side网页图标更改问题

### DISCUSSION THREAD

**Discussion by Ssbx-xsb** [dotnetcore/BootstrapBlazor#3510] [Q&A] [answered]
### Which class is this unit test associated with?

模版项目Server-Side没有了mvc那些cshtml默认文件，之前在_Layout里面可以更改图标并支持svg格式，现在我在app.razor里面更改图标发现不支持svg格式。
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/49543991/26bb4aaa-1940-4dd7-adfa-0781c7373256)

**Comment by bb-auto** [comment]
@Ssbx-xsb Thank you for reporting. We will give triage later.

**Reply by ArgoZhang** [reply, marked as answer]
支持的，你没弄对，去看看基础先，`~/` 这种肯定不行的，右上角帮忙点一下 `Star` 谢谢啦

**Comment by Ssbx-xsb** [comment]
这里选择智能提示里面的就带了~/，去掉后好用了
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/49543991/78615937-ee0e-4ca7-97b7-d2e18bf65934)
